tests/acceptance/01-InstallJoomlaCept.php

```diff
@@ -9,7 +9,7 @@
 // Load the Step Object Page
 $I = new \AcceptanceTester($scenario);
 
-$I->wantToTest('redCORE installation in Joomla 3');
+$I->wantToTest('Joomla 3 Installation');
```

Comment:
you are installing redCORE twice. Please remove it from 01-InstallJoomlaCept leaving it only in 02-InstallExtensionCept